namespace Define.Common.Exceptions;
public class MatchModelLoggerMessages
{
    public const string DeleteException = "The delete function on Match Model is not succeed";
    public const string DeleteSuccess = "The delete function on Match Model is succeed";

    public const string GetAllMatchModelException = "The get all function on Match Model is not succeed";
    public const string GetAllMatchModelSuccess = "The get all function on Match Model is succeed";

    public const string GetByIdMatchModelException = "The get by ID function on Match Model is not succeed";
    public const string GetByIdMatchModelSuccess = "The get by ID function on Match Model is succeed";

    public const string SaveMatchModelException = "The save function on Match Model is not succeed";
    public const string SaveMatchModelSuccess = "The save function on Match Model is succeed";
}
